examples/soap-calculator/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import {SoapCalculatorApplication} from './application';
+import {ApplicationConfig} from '@loopback/core';
+import {RestServer} from '@loopback/rest';
+
+export async function main(options?: ApplicationConfig) {
+  const app = new SoapCalculatorApplication(options);
+  await app.boot();
+  await app.start();
+
+  const server = await app.getServer(RestServer);
+  const port = await server.get<number>('rest.port');
+  console.log(`Server is running at http://127.0.0.1:${port}`);
```

Comment:
I would like us to simplify this part of app setup. I opened a pull request to make that happen - see https://github.com/strongloop/loopback-next/pull/1572. If my PR lands before yours, then this part will need an update.

Reply:
#1572 has landed. Line 10-13 can be removed and replaced with: 

```
const url = app.restServer.url;
console.log(`Server is running at ${url}`);
```